src/components/RectangularButton.tsx

```diff
@@ -1,11 +1,23 @@
 import * as React from "react";
-import { StyleSheet, View, Text, Pressable, GestureResponderEvent, ViewStyle } from "react-native";
+import {
+  StyleSheet,
+  View,
+  Text,
+  Pressable,
+  GestureResponderEvent,
+  ViewStyle,
+} from "react-native";
 
-const RectangularButton = ({ text, onPress, buttonStyle, textStyle }: {
-  text: string,
-  onPress: (event: GestureResponderEvent) => void,
-  buttonStyle: any,
-  textStyle: any,
+const RectangularButton = ({
+  text,
+  onPress,
+  buttonStyle,
+  textStyle,
+}: {
+  text: string;
+  onPress: (event: GestureResponderEvent) => void;
+  buttonStyle: any;
+  textStyle: any;
 }) => {
   const styles = StyleSheet.create({
```

Comment:
I know its v unnecessary but let's just do separate file to be consistent

Reply:
That way people who use the codebase know exactly what to expect when they're looking for styling